pkg/portal/graphql/context.go

```diff
@@ -107,6 +107,7 @@ type TutorialService interface {
 
 type OnboardService interface {
 	SubmitOnboardEntry(entry model.OnboardEntry, actorID string) error
+	CheckCompletion(actorID string) (bool, error)
```

Comment:
Lets be clear what we completed on the method name: `CheckOnboardingSurveyCompletion`.